### DISCUSSION TITLE
Increase data limit

### DISCUSSION THREAD

**Discussion by sharpi123** [absadiki/subsai#58] [Q&A]
Hello. I am converting a movie, and the file size is over 200 megs. I get a message telling me that I can increase the limit, but I don't quite understand how. What steps would I have to follow to be able to do this?

**Comment by absadiki** [comment]
Hello @sharpi123, 

You can increase the limit  (for example to 500 MB) by running:
```shell
 subsai-webui --server.maxMessageSize 500
```
But I think in your case (a movie will exceed 1GB probably) the webui will freeze and or may crash and it's not efficient. 

I would suggest to use the webui on a fragment of the file just to adjust the configs (like the model name, model size, etc) and once you get the prefect set of configurations depending on your machine capabilities, use the command line interface (CLI) to transcribe your larger files.

**Reply by sharpi123** [reply]
Hello again. I have tried the command, but it keeps failing me. At one point the process stops, therefore it never ends. I have the latest version, and my file weighs 300 MB, it is an audio described movie from which I am precisely extracting the audio description. Is there any way, changing the files, to set a value other than 200 MB?

**Reply by absadiki** [reply]
what do you mean by it keeps failing ? You mean even after executing the command above, it still giving you the 200 mb constraint ?

**Reply by sharpi123** [reply]
Exactly. When I run the comand it still limits me to 200 MB. With an older version it gives the error, with the latest one it does not give it but still disconects in a moment of the process.

**Reply by absadiki** [reply]
Does it disconnect without any errors ? 
Have you tried to command line ? Does it disconnect as well ?

**Reply by sharpi123** [reply]
Connection error
Is Streamlit still running? If you accidentally stopped Streamlit, just restart it in your terminal:
$"python -m streamlit" run C:\Users\luchi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\subsai\webui.py --theme.base dark C:\Users\luchi\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts\subsai-webui
Done
Close

**Reply by absadiki** [reply]
Weird .. I don't see from the error that the you ran the command with `server.maxMessageSize`!
If you can share the audio file, I will give it a try on my end ?

**Comment by albertputinjobs** [comment]
The server.maxMessageSize command doesn't work it is still showing 200mb limit only.

**Comment by albertputinjobs** [comment]
go to this folder C:\Users\Z\.streamlit and create config.toml with contents as below and that's it.


[server]
maxUploadSize = 4500

maxMessageSize = 9999
